common/web/input-processor/tests/cases/predictionContext.js

```diff
@@ -285,7 +285,7 @@ describe("PredictionContext", () => {
     let returnValue = predictiveContext.accept(reversion);
 
     // 'accepting' a reversion performs a rewind; there's no need for async ops here.
-    assert.isNull(returnValue); // as per the method's spec.
+    assert.isNull(await returnValue); // as per the method's spec.
```

Comment:
It would be cleaner to put the await in the original call rather than bury it in the assertion I think.

```
 let returnValue = await predictiveContext.accept(reversion);
```